virttest/utils_misc.py

```
@@ -4698,3 +4695,79 @@ def _md5(fd):
+        raise exceptions.TestFail(
+            f"CVM state is {guest_state or 'missing'}, expected 'running'"
+        )
+    LOG.info(f"Verifying cvm {cvm_type} capability enablement in guest")
+    guest_check_cmd = params["cvm_guest_check"]
+    try:
+        return_code, output = session.cmd_status_output(guest_check_cmd, timeout=240)
+        if return_code != 0:
+            raise exceptions.TestFail(
+                f"Guest cvm {cvm_type} capability check failed with "
+                "return code {return_code}: {output}"
+            )
```

Ruff (0.14.8)

Abstract `raise` to an inner function

(TRY301)